src/TextUI/Command.php

```diff
@@ -84,6 +84,7 @@ class Command
         'disallow-test-output'      => null,
         'disallow-resource-usage'   => null,
         'disallow-todo-tests'       => null,
+        'default-time-limit='       => null,
```

Comment:
For consistency, I would rather not add an option like `--default-time-limit` to set the default time limit on the CLI. The timeouts for `@small`, `@medium`, and `@large` can also only be configured in the configuration file.

Reply:
Yes, that is true. I asked @reinholdfuereder what the exact use case would be and this time limit is meant to be for tests that are _not marked_ with a `@size`: https://github.com/sebastianbergmann/phpunit/issues/2085#issuecomment-405539959

I know a bunch of older test collections with legacy would benefit from this.

Reply:
Not sure about the trailing`=`, what do you think?

Reply:
The `=` is required for `--key=value`, right? I have not found a clean way of doing `--key[=value]`.

Reply:
The `=` means that the option requires a value.

Reply:
Thank you for clarifying, @sebastianbergmann!